sections/Product/FeaturedGallery.tsx

```diff
@@ -0,0 +1,58 @@
+import type { ImageWidget } from "apps/admin/widgets.ts";
+import Image from "apps/website/components/Image.tsx";
+import type { Product } from "apps/commerce/types.ts";
+import ProductCard from "../../components/product/ProductCard.tsx";
+export interface Props {
+    title: string;
+    idProduct: Product[] | null;
+    featuredPhoto: {
+        src: ImageWidget;
+        alt: string;
+        href: string;
+    },
+    /**
+     * @title reversed order?
+     */
+    directionInverse: {
+        /** @default "col" */
+        mobile?: "col" | "col-reverse";
+        /** @default "row" */
+        desktop?: "row" | "row-reverse";
+    };
+}
+
+const MOBILE_DIRECTION = {
+    "col": "flex-col",
+    "col-reverse": "flex-col-reverse",
+};
+
+const DESKTOP_DIRECTION = {
+    "row": "lg:flex-row",
+    "row-reverse": "lg:flex-row-reverse",
+};
+
+export default function FeaturedGallery({ title, featuredPhoto, directionInverse, idProduct }: Props) {
+
+    if (!idProduct || idProduct.length === 0) {
+        return null;
+    }
+
+    return (
+        <div class="flex px-4 flex-col gap-2 py-2">
+            <h3 class=" text-base text-left uppercase font-bold">
```

Comment:
segue a ordem de heading: so chama h3 se voce ja chamou um h2 em um elemento pai
```suggestion
            <h2 class=" text-base text-left uppercase font-bold">
```